scripts/seed-database/write/teams.sql

```diff
@@ -3,6 +3,7 @@ CREATE TEMPORARY TABLE sync_teams (
   id integer,
   name text,
   slug text,
+  -- TODO: Drop this and fetch from team_themes
```

Comment:
TODO intentionally left here, we'll need to update sync scripts only once this new table is in prod.

Reply:
Hey @DafyddLlyr 

I think this _might_ be related but I was having issues building `main` since coming back from the holidays. 

I was seeing: 

```
psql:write/teams.sql:18: ERROR:  invalid input syntax for type json
DETAIL:  Expected end of input, but found "-11".
```

Which was resolved by commenting out `theme jsonb,`